dist/Renderer.js

```diff
@@ -0,0 +1,20 @@
+class Renderer {
+    constructor() {
+    }
+
+    renderAdd() {
+        $("#container").empty()
+        $("#total").remove()
+        const addTemplate = Handlebars.compile($("#add-template").html())
+        $("#container").append(addTemplate())
```

Comment:
nicer way would be - `$("#container").empty().append(addTemplate())` - and delete line #6